markets/bfp-market/test/integration/modules/PerpMarketFactoryModule.utilisation.test.ts

```diff
@@ -229,33 +220,27 @@ describe('PerpMarketFactoryModule Utilization', () => {
         minDelegationD18,
         bn(1)
       );
-      const tradersGenerator = toRoundRobinGenerators(shuffle(traders()));
-      const trader1 = tradersGenerator.next().value;
-      const trader2 = tradersGenerator.next().value;
 
       // Create one trade that will win more than the delegated collateral
-      const { collateral: collateral1, collateralDepositAmount: collateralDepositAmount1 } =
-        await depositMargin(bs, genTrader(bs, { desiredMarket: market, desiredTrader: trader1 }));
+      const { trader, collateral, collateralDepositAmount } = await depositMargin(
+        bs,
+        genTrader(bs, { desiredMarket: market })
+      );
 
       // Create a long position
-      const order1 = await genOrder(bs, market, collateral1, collateralDepositAmount1, {
+      const order1 = await genOrder(bs, market, collateral, collateralDepositAmount, {
         desiredSide: 1,
       });
-      await commitAndSettle(bs, marketId, trader1, order1);
+      await commitAndSettle(bs, marketId, trader, order1);
 
       // Price 10x, causing large profits for the trader
       const newMarketOraclePrice = wei(order1.oraclePrice).mul(10).toBN();
       await market.aggregator().mockSetCurrentPrice(newMarketOraclePrice);
-      const closeOrder1 = await genOrder(bs, market, collateral1, collateralDepositAmount1, {
-        desiredSize: order1.sizeDelta.mul(-1),
+      const decreaseOrder1 = await genOrder(bs, market, collateral, collateralDepositAmount, {
+        desiredSize: wei(order1.sizeDelta).mul(-1).mul(0.9).toBN(),
       });
-      await commitAndSettle(bs, marketId, trader1, closeOrder1);
 
-      // Create a new position with a different trader. This trader will will not incur and profits or losses.
-      const { collateral: collateral2, collateralDepositAmount: collateralDepositAmount2 } =
-        await depositMargin(bs, genTrader(bs, { desiredMarket: market, desiredTrader: trader2 }));
-      const order2 = await genOrder(bs, market, collateral2, collateralDepositAmount2);
-      await commitAndSettle(bs, marketId, trader2, order2);
```

Comment:
Had to change this test as this order would now fail with the new minimumCredit requirement.